schema.sql

```diff
@@ -1,5 +1,15 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id serial primary key,
+	name varchar(50),
+	date_of_birth date,
+	escape_attempts INT,
+	neutered boolean,
+	weight_kg decimal
 );
+
+-- add species by updating table animal
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
```

Comment:
Remove column species

Add column species_id which is a foreign key referencing species table
Add column owner_id which is a foreign key referencing the owners table

- [ ] Kindly work on the schema

Reply:
Thanks for the review is it okay now 


> Remove column species
> 
> Add column species_id which is a foreign key referencing species table Add column owner_id which is a foreign key referencing the owners table
> 
> * [ ]  Kindly work on the schema

Is it okay now

Reply:
Is the Issue resolved now ?